index.ts

```diff
@@ -59,12 +59,41 @@ const plugin: Plugin<ReplicatorMetaInput> = {
         }
 
         if (batch.length > 0) {
-            await fetch(`https://${config.host.replace(/\/$/, "")}/e`, {
+            const batchDescription = `${batch.length} event${batch.length > 1 ? 's' : ''}`
+            await fetch(`https://${config.host.replace(/\/$/, '')}/e`, {
                 method: 'POST',
                 body: JSON.stringify(batch),
                 headers: { 'Content-Type': 'application/json' },
-            })
-            console.log(`Flushing ${batch.length} event${batch.length > 1 ? 's' : ''} to ${config.host}`)
+            }).then(
```

Comment:
I think we can just use `try/catch` here instead of `.then`?

I don't know which is more desirable tbh.